frontend/src/nav/Menu.js

```diff
@@ -1,15 +1,28 @@
 import styled from "styled-components/macro";
-import logo from "../pictures/logo.png";
+import logo from "../images/logo.png";
+import { NavLink } from "react-router-dom";
 
-export default function Menu({ open }) {
+export default function Menu({ open, setOpen }) {
```

Comment:
I think it's a better solution to define `setOpen` where you pass the function as prop. 
In file where `Menu` is used:

`<Menu setOpen={() => setOpen(!open)}/>`